styleguide.config.js

```diff
@@ -43,6 +43,10 @@ module.exports = {
         {
           name: 'Full Screen Dropzone',
           content: 'examples/Fullscreen/Readme.md'
+        },
+        {
+          name: 'Dragging folders example',
```

Comment:
Keeping in mind my first comment I'm wondering if this should become a more generic name. Like "Extending Dropzone"? Or "Custom `getDataTransferItems`"?

Reply:
Yes, I agree, one of those names would be more suitable. But at the same time I think people are looking for solving specific problem when they are reading docs so I would suggest making a section “Extending Dropzone” and introduce some subsections to it like the one with Dropzone that accepts folders. I deem it more logical while all the value of libraries lies within the simplicity of use in every particular use-case, maybe after we can also think of other examples and add more subsections to make the info flow more pleasant. What do you think?

Reply:
I think `Extending Dropzone` as a separate page and then sub-sections with different examples could work best.